.github/workflows/base.yml

```diff
@@ -0,0 +1,30 @@
+name: Base image build
+
+on: 
+  workflow_dispatch:
+    branches: 
+     - '*'
+     - '!flatcap'
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1  
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1 
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner  }}
+        password: ${{ secrets.GH_TOKEN }}
```

Comment:
Is this a user-defined secret? The docs recommend using `GITHUB_SECRET` for authentication[0]. That looks like a less error-prone authentication method.

[0] https://docs.github.com/en/actions/reference/authentication-in-a-workflow

Reply:
It is the same thing, you have to get this token from GH with needed access rights and put into repository secrets.

Reply:
Yes.